ponyup-init.sh

```diff
@@ -27,6 +27,17 @@ BLUE="\e[34m"
 RED="\e[31m"
 YELLOW="\e[33m"
 
+shasumCommand() {
+  if command -v sha256sum > /dev/null 2>&1; then
+    sha256sum "$@"
+  elif command -v shasum > /dev/null 2>&1; then
+    shasum --algorithm 256 "$@"
+  else 
+    echo "No checksum command!"
```

Comment:
can you update this to print in RED and use printf instead of echo like:

`printf "%bNo checksum command found.%b\n"` "${RED}" "${DEFAULT}"`